.github/actions/setup-xcode/action.yml

```diff
@@ -0,0 +1,14 @@
+name: Setup xcode
+description: 'Set up your GitHub Actions workflow with a specific version of xcode'
+inputs:
+  xcode-version:
+    description: 'The xcode version to use'
+    required: false
+    default: '14.3.1'
+runs:
+  using: "composite"
+  steps:
+    - name: Setup xcode
+      uses: maxim-lobanov/setup-xcode@v1.6.0
```

Comment:
> `maxim-lobanov`/ ? Is this an action owned by GH or by a third-party dev?  

This is not owned by GH. It's a 3rd party action provided by a community member. It is open source.

Reply:
cc. @cortinico which policies do we follow, in this case? Is it ok to use this? I'm concern because, at some point, we will have to start signing Hermes in CI and if this is not managed by us or GH directly, it means that a contributor might inject malicious code and we might ship malicious code to all our users.

Reply:
I guess we could go ahead with this for now.
The follow-up would be to fork https://github.com/maxim-lobanov/setup-xcode
inside gh/react-native-community

Reply:
The best option to address your concern is to pin to a commit sha as follows...
```
    - name: Setup xcode
      uses: maxim-lobanov/setup-xcode@60606e260d2fc5762a71e64e74b2174e8ea3c8bd
```

Reply:
@robandpdx could you pin the action with the commit, for the time being?
After this small change, I'll import the PR and I'll try to land it!

Reply:
@cipolleschi Done ✅